src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1792,6 +1793,9 @@ private static MeshGeometry3D DrawTestMesh()
 
         internal void UpdateNearClipPlane()
         {
+
+            if (camera == null) return;
```

Comment:
for every mouse up on watch view, updatenearclip... is called. is that watch view is watch3d?